models/AbstractModel.php

```diff
@@ -6,5 +6,20 @@
      * Base class for implementing other Models
      */
     abstract class AbstractModel {
-        
+
+        /**
+         * Table name to which the model belongs
+         *
+         * @var string
+         */
+        private $tableName;
+
+        /**
+         * Returns table name
+         *
+         * @return string
+         */
+        public function getTableName():string {
+            return $this->tableName;
+        }
```

Comment:
Added `tableName` property and `getTableName` method so that it actually exists.